final/src/main/java/ua/epam/spring/hometask/dao/impl/EventDAOImpl.java

```diff
@@ -0,0 +1,78 @@
+package ua.epam.spring.hometask.dao.impl;
+
+import org.springframework.stereotype.Repository;
+import ua.epam.spring.hometask.dao.EventDAO;
+import ua.epam.spring.hometask.domain.Event;
+import ua.epam.spring.hometask.util.EventIdIncrementator;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Repository
+public class EventDAOImpl implements EventDAO {
+
+    private static Set<Event> events;
+
+    public void setEvents(Set<Event> eventsSet) {
+        events = eventsSet;
+    }
```

Comment:
My personal preference is putting setters and getters at the end of the class. So it makes the code more readable as first you see API methods.